sklearn/metrics/tests/test_classification.py

```diff
@@ -215,6 +215,46 @@ def test_classification_report_zero_division_warning(zero_division):
             assert not record
 
 
+def test_classification_report_input_subset_of_labels():
+    y_true, y_pred = [0, 1], [0, 1]
```

Comment:
I think that we want a slightly different test isntead: we want that different input in `labels` show or not the "accuracy". So we could parametrize the test by trying `labels=([0, 1, 2], [0, 1], [0])`.